monitoring/rid_qualifier/flight_data_generator.py

```diff
@@ -0,0 +1,360 @@
+from shapely.geometry import Point, Polygon
+import shapely.geometry
+from pyproj import Geod, Transformer
+import json
+from pathlib import Path
+from typing import List, NamedTuple
+import arrow
+import datetime
+from datetime import datetime, timedelta
+import uuid
+
+class QueryBoundingBox(NamedTuple):
+    ''' This is the object that stores details of query bounding box '''
+    name: str
+    shape: Polygon
+    timestamp_before: timedelta
+    timestamp_after: timedelta
+    
+class FlightPoint(NamedTuple):
+    ''' This is the object that stores details of query bounding box '''
+    lat: float
+    lng: float
+    alt: float
```

Comment:
Can we specify the units and reference frame expected for this field so we can verify whether logic which writes and reads it is correct?  For instance, it appears to have meters AGL assigned into it, but maybe meters WGS84 expected from it later?

Reply:
Question: What do you mean by reference frame?

Reply:
Also is there merit in having altitude AGL vs WGS84? Since the DSS uses WGS84 (but many UTM software / Ground control stations) use AGL. Do you have a preference? The current implementation is using Bern, I can do a conversion to WGS84 for the area but perhaps AGL is easier to understand?

Reply:
OK I have made a first pass at it, perhaps we need to move it out as a separate method. I know the basics of Geoid / ellipsoid heights but happy to learn / correct as necessary.

Reply:
OK I have now completed a first pass at it.